# Study 961 — Which Gold 🪙

**Five wrappers, one metal. Does the cheap one actually keep more of it?**

**GLD, IAU, GLDM, SGOL, BAR** all hold allocated London bullion in a vault, publish a bar
list, and strike at the same 16:00 New York close. They differ in one contractual respect:
the sponsor fee, from **10 bp** (GLDM) to **40 bp** (GLD). If the fee sheet is a real number
rather than a marketing one, the ranking of what these five actually delivered should be the
exact reverse of the ranking of what they charge.

Common window **2018-06-26 → 2026-06-30** (2,013 sessions, 96
complete months). None of the five pays a distribution, so their adjusted close *is* their
price close.

*Numbers below are the frozen headline (`docs/results.md`, Fingerprint `9af52c692ea9`); the live
cells run the offline synthetic control and are labelled as such. As-of 2026-06-30.*


## 1. The setup — the only thing that separates them is the price tag

There is no skill question here, and no market view. Nobody is forecasting gold. The five trusts own the same metal; the sponsor takes its cut by quietly selling a sliver of bullion. So the question is arithmetic: **does the sliver show up?**

In [1]:
R = {'fees': {'GLD': 40.0, 'IAU': 25.0, 'GLDM': 10.0, 'SGOL': 17.0, 'BAR': 17.49}}
ORDER = ['GLDM', 'SGOL', 'BAR', 'IAU', 'GLD']
print('wrapper   fee (bp/yr)')
for t in ORDER:
    print('  %-6s %6.2f' % (t, R['fees'][t]))
print('\nfee spread across the cohort: %.0f bp/yr' % (max(R['fees'].values()) - min(R['fees'].values())))

wrapper   fee (bp/yr)
  GLDM    10.00
  SGOL    17.00
  BAR     17.49
  IAU     25.00
  GLD     40.00

fee spread across the cohort: 30 bp/yr


## 2. The answer — the ranking is perfect

Below: each wrapper's realised return minus the average of all five, in basis points a year, over eight years. Read the two columns side by side.

In [2]:
R = {'fees': {'GLD': 40.0, 'IAU': 25.0, 'GLDM': 10.0, 'SGOL': 17.0, 'BAR': 17.49}, 'td': {'GLD': -16.75, 'IAU': -2.24, 'GLDM': 9.43, 'SGOL': 5.41, 'BAR': 4.15}, 'td_t': {'GLD': -3.02, 'IAU': -0.31, 'GLDM': 1.8, 'SGOL': 0.56, 'BAR': 0.54}, 'td_up': {'GLD': 32, 'IAU': 47, 'GLDM': 59, 'SGOL': 49, 'BAR': 51}, 'spearman': -1.0}
ORDER = ['GLDM', 'SGOL', 'BAR', 'IAU', 'GLD']
print('wrapper   fee    what it actually delivered vs the cohort average')
for t in ORDER:
    print('  %-6s %5.2f   %+7.2f bp/yr   (t = %+5.2f, up in %d of 96 months)'
          % (t, R['fees'][t], R['td'][t], R['td_t'][t], R['td_up'][t]))
print('\nrank correlation between fee and outcome: %.4f  (a perfect inversion)' % R['spearman'])

wrapper   fee    what it actually delivered vs the cohort average
  GLDM   10.00     +9.43 bp/yr   (t = +1.80, up in 59 of 96 months)
  SGOL   17.00     +5.41 bp/yr   (t = +0.56, up in 49 of 96 months)
  BAR    17.49     +4.15 bp/yr   (t = +0.54, up in 51 of 96 months)
  IAU    25.00     -2.24 bp/yr   (t = -0.31, up in 47 of 96 months)
  GLD    40.00    -16.75 bp/yr   (t = -3.02, up in 32 of 96 months)

rank correlation between fee and outcome: -1.0000  (a perfect inversion)


Cheapest first, priciest last, in exactly the order the fee sheet predicts, with **no inversions**. The gap between the ends — GLDM's 10 bp against GLD's 40 bp — is worth **+26.17 basis points a year** on the tape, against the **30 bp** the fee sheets promise: *t* = **+3.18**, positive in **65** of 96 months and in **9 of 9** calendar years, with a 95% interval [+9.5, +41.9] that never touches zero.

Note how wide that interval is. The tape says *somewhere between 10 and 42 basis points* — not 26 on the nose. What it does not say is zero.

> 🔬 **For the quants:** the estimator is the mean of non-overlapping complete-calendar-month log differences, annualised. The *t* above is the plain iid one **on purpose**: this residual mean-reverts (acf1 = -0.42 monthly, -0.46 daily), so a Newey-West *t* comes out *larger* (+6.32) rather than smaller, and larger still as you widen the bandwidth. We quote the small one. The daily view of the same effect is powerless either way: naive *t* = +0.78.

## 3. What it costs to own the wrong one

No turnover, no timing, no decision after the first one. The gap simply compounds.

In [3]:
R = {'cost1': 0.262, 'cost5': 1.302, 'cost10': 2.587, 'cost20': 5.107}
for y, pct in [(1, R['cost1']), (5, R['cost5']), (10, R['cost10']), (20, R['cost20'])]:
    print('%2d years of the dear wrapper: %.3f%% of your gold sleeve, or $%6s per $100,000'
          % (y, pct, format(int(round(pct/100*100000)), ',')))

 1 years of the dear wrapper: 0.262% of your gold sleeve, or $   262 per $100,000
 5 years of the dear wrapper: 1.302% of your gold sleeve, or $ 1,302 per $100,000
10 years of the dear wrapper: 2.587% of your gold sleeve, or $ 2,587 per $100,000
20 years of the dear wrapper: 5.107% of your gold sleeve, or $ 5,107 per $100,000


Ten years of holding GLD instead of GLDM costs about **2.6% of the sleeve** — roughly $2,587 on $100,000. That is the whole prize. It is small, it is certain, and it requires nothing of you but reading a fee sheet once.

## 4. The catch — three of them

**(a) You cannot fix this by switching.** Selling an appreciated GLD position triggers tax (US bullion trusts are taxed as collectibles). At a 20% rate on a position that has doubled, the 26 bp gap needs **26 years** to repay the bill; at 23.8% on a position up 4×, **75 years**. This is a rule for money you are *about to* invest.

**(b) Cheapest is not always buyable.** The fee-cheapest tier is also the thinnest. A $100m ticket is 0.02 days of GLD's volume and **4.8 days** of BAR's. GLDM, at $433m a day, is the one place where cheap and liquid coincide.

**(c) The fine print of the ladder is unreadable.** The tape can separate a 30 bp gap. It cannot separate a 7 bp one: GLDM against SGOL, SGOL against BAR — all statistically nothing. The ranking is right at the top and noise at the rungs.

## 5. Chasing last year's best tracker does not work

The obvious next thought — buy whichever wrapper tracked best last year — is worse than doing nothing. Ranked at each year-end, filled at the next session's close, 2 bp of cost per switch: **0.7762** excess Sharpe against **0.7799** for simply owning the cheapest (4 switches over eight years, *t* on the difference = -0.14). The fee is the signal; last year's tracking is not.

## 6. Live check — the machinery is unbiased (offline synthetic)

**This cell is synthetic, not the real tape.** On a *planted* world where five wrappers really are shaved by their published fees, the stack must recover the ladder. On a null where every wrapper charges the same fee — while the published sheet still shows dispersion — it must find nothing.

In [4]:
import os, sys
sys.path.insert(0, os.path.abspath('..'))
sys.path.insert(0, os.path.abspath(os.path.join('..','..','..')))
import numpy as np
from which_gold import data, strategy as st
pl_p, pl_t = data.synthetic_panel(signal_strength=1.0, seed=961)
pl = st.synthetic_detect(pl_p, pl_t)
print('SYNTHETIC planted (gap %.1f bp/yr): recovered %+.2f bp/yr, t=%+.2f, rank corr %+.3f'
      % (pl['planted_gap_bpy'], pl['pair_spread_bpy'], pl['pair_t'], pl['spearman']))
nl = [st.synthetic_detect(*data.synthetic_panel(signal_strength=0.0, seed=961+s)) for s in range(8)]
sp = np.array([n['pair_spread_bpy'] for n in nl]); tt = np.array([n['pair_t'] for n in nl])
print('SYNTHETIC null x8: spread mean %+.2f bp/yr (sd %.2f), |t|>=2 in %d/8 (should be 0)'
      % (sp.mean(), sp.std(ddof=1), int((abs(tt) >= 2).sum())))

SYNTHETIC planted (gap 30.0 bp/yr): recovered +30.98 bp/yr, t=+2.55, rank corr -1.000


SYNTHETIC null x8: spread mean -0.47 bp/yr (sd 1.07), |t|>=2 in 0/8 (should be 0)


## Verdict

- **Signal — Real.** The cheapest wrapper beat the priciest by **+26.17 bp/yr** with *t* = **+3.18**, an interval [+9.5, +41.9] clear of zero, the same sign in **9 of 9** calendar years and in **both** halves of the sample, and the five-wrapper ranking is a **perfect** inversion of the fee ranking. Where the fee sheet is coarse the tape confirms it; where it is fine (7 bp apart) the tape cannot see it — and one wrapper at a time, only the priciest separates from the pack on its own.
- **Tradability — Investable.** It is a purchase decision, not a strategy: no forecast, no turnover, no capacity limit on the wrapper that wins, and **2.6% of a ten-year gold sleeve** at stake. Buy the cheap one with new money and never trade it — do not sell an appreciated holding to get there, do not try to short the dear one, and do not chase last year's tracker.